src/Raku/ast/code.rakumod

```diff
@@ -193,7 +194,8 @@ class RakuAST::Code
         # fixed up as needed.
         $block.code_object($code-obj);
 
-        my @compstuff := nqp::getattr($code-obj, Code, '@!compstuff');
+        # XXX: @!compstuff can be undefined during an EVAL?
+        my @compstuff := nqp::getattr($code-obj, Code, '@!compstuff') // [];
```

Comment:
If `@!compstuff` is null that's usually because we didn't run IMPL-STUB-CODE. That indicates a bug in the compiler. This change would hide such problems. You need to find out where this RakuAST::Code object gets created and how we end up in IMPL-LINK-META-OBJECT before IMPL-STUB-CODE could run. As always in RakuAST, it's about timing.